color/p5js/Law of simmilarity + color/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: [p5js]
+author: [Daniele Maselli]
```

Comment:
No `[]` for the author. See https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md#p5js